image/php-fpm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu
+
+# ADD /laravel /var/www
```

Comment:
Also: https://github.com/atrauzzi/docker-laravel/blob/master/images/data/build#L17 anywhere it says `CONTEXT_ROOT`.